src/components/ProfilePage.js

```diff
@@ -98,83 +100,263 @@ export default function ProfilePage() {
+                            {projectFollowings.map((projectFollowing) => {
+                                return (
+                                    <Card>
+                                    <Card.Body>
+                                        <Card.Title>{projectFollowing.projectName}</Card.Title>
+                                        <Card.Text>
+                                        The project id is: {projectFollowing.projectId}
+                                        </Card.Text>
+                                        <Card.Link href={'/projects/' + projectFollowing.projectId}>Link to project</Card.Link>
+                                    </Card.Body>
+                                    </Card>
+                                );
+                            })
+                            }
+                            </div>
                         </Col>
-                        <Col>
-                            <Form>
-                            <Form.Group className="mb-3" controlId="exampleForm.ControlTextarea1">
-                            <Form.Label>Contact Information</Form.Label>
-                            <Form.Control as="textarea" rows={3} defaultValue = {links} readOnly/>
-                            </Form.Group>
-                            </Form>
+                        <Col xs lg="4">
+                            <h1 className="text-info">Project Membership</h1>
+                            <div className="project-membership-scroll">
+                        
+                            {projectMemberships.map((projectMembership) => {
+                                return (
+                                    <Card style={{ width: '18rem' }}>
+                                    <Card.Body>
+                                        <Card.Title>{projectMembership.projectName}</Card.Title>
+                                        <Card.Subtitle className="mb-2 text-muted">{projectMembership.projectStatus}</Card.Subtitle>
+                                        <Card.Text>
+                                        The project id is: {projectMembership.projectId}
+                                        </Card.Text>
+                                        <Card.Link href={'/projects/' + projectMembership.projectId}>Link to project</Card.Link>
+                                    </Card.Body>
+                                    </Card>
+                                    );
+                                })
+                                }
+                            </div>
                         </Col>
                     </Row>
                     </div>
                 )
             }
-            {
-                canEdit && (
-                    <div>
+            {canEdit && (
+                <div>
+                   <Row>
+                        <Carousel fade>
+                            <Carousel.Item>
```

Comment:
Since the Carousel appears regardless of `canEdit`, the code didn't have to be copied here again, but rather shown always, even if the user is a guest user.